projects/fsastorefrontlib/src/cms-components/checkout/components/bind-quote-dialog/bind-quote-dialog.component.ts

```diff
@@ -40,6 +40,7 @@ export class BindQuoteDialogComponent {
 
   bindQuote() {
     this.quoteService.bindQuote(this.cartCode);
+    localStorage.setItem('bindingState', JSON.stringify(true));
```

Comment:
Checking if quote is bound via local storage doesn't seem very reliable to me. What happens if user clears the browser cache or uses another browser/computer to access his/hers account? Accessing via different devices (phone, tablet...) seems like a viable scenario to me.

Reply:
Yes, in those cases it will be lost. Maybe I could try similar with `BehaviorSubject`

Reply:
Not sure if that would help either, bcs it will also keep the state on client-side, in the browser. Maybe we need to have some brain-storming session about this.